python/format/src/datasets.py

```diff
@@ -46,8 +46,9 @@ class Validator:
     def run_static_analysis(self):
         try:
             file_path, self.file = _load_file(self.file_or_file_path)
-            rdf_graph = graphs.load_rdf_graph(self.file)
-            nodes = graphs.check_rdf_graph(self.issues, rdf_graph)
+            rdf_graph, rdf_nx_graph = graphs.load_rdf_graph(self.file)
+            rdf_namespace_manager = rdf_graph.namespace_manager
```

Comment:
Note to the reviewer: I'm maybe thinking of using this namespace manager to refactor how I parse the JSON. I could go from JSON to our graph representation without using the intermediary RDF graph.

Reply:
SGTM